test/test_helper.exs

```diff
@@ -1 +1,4 @@
 ExUnit.start(exclude: [:pending])
+{:ok, _} = Application.ensure_all_started(:wallaby)
+Application.put_env(:wallaby, :base_url, ElixirConsoleWeb.Endpoint.url())
+Application.put_env(:wallaby, :screenshot_on_failure, true)
```

Comment:
Could this be placed in config/test? That's probably where I'd expect to find it

Reply:
Good call.

I think we can move `screenshot_on_failure` setting to `config/test.exs`, but apparently not the `base_url` because `ElixirConsoleWeb.Endpoint.url` not available yet at that point.

I guess that would explain why the recommend the `test_helper.exs` for that particular setting in https://github.com/elixir-wallaby/wallaby#phoenix.